open-craft/opencraft
# BB-2083: Add deployment toolbar + modals, fix tests and lint code

This PR implements and connects the redeployment toolbar to Ocim backend.

**Testing instructions:**
1. Checkout this branch. Run Ocim backend.
2. On another console `cd frontend/` and run the new UI `npm run start`.
3. Create a new betatest instance using the form.
4. Click login and enter your credentials.
5. Wait for the instance to provision (or force the appserver to an active state).
6. Check that the redeployment toolbar is working as expected.
7. Change the instance name and email.
8. Check that the deploy button was enabled.
9. Click deploy.
10. Check that the deploy button got disabled and that now you have an option to cancel the redeployment.
11. Cancel the redeployment.
12. Deploy again, and wait until it's provisioned.
13. Check code quality & implementation.

This is currently running on stage, so feel free to test there too.

**Reviewer:**
